app/views/admin/conference_domains/show.haml

```diff
@@ -17,14 +17,15 @@
           %td
             = @conference.custom_domain
           %td
-            - ck_domain = ConferenceDomainsService.new(conference: @conference).check_custom_domain
-            - if ck_domain == true
-              %span.glyphicon.glyphicon-ok
-            - elsif ck_domain == false
-              %span.glyphicon.glyphicon-remove
+            - if ENV['OSEM_HOSTNAME'].present?
+              - ck_domain = ConferenceDomainsService.new(@conference).check_custom_domain
+              - if ck_domain
+                %span.glyphicon.glyphicon-ok
+              - else
+                %span.glyphicon.glyphicon-remove
             - else
               %i
-                = ck_domain
+                feature disabled
```

Comment:
Sure, but if people see "feature disabled" here, they might not know why it's disabled, and they might not realize how to enable it. So you can link to that wiki page from here then.